online-inference/custom-sentiment/sentiment-inferenceservice.yaml

```diff
@@ -1,29 +1,33 @@
-apiVersion: serving.kubeflow.org/v1alpha2
+apiVersion: serving.kubeflow.org/v1beta1
 kind: InferenceService
 metadata:
-  annotations:
-    serving.kubeflow.org/gke-accelerator: Tesla_V100
   labels:
     qos.coreweave.cloud/latency: low
   name: sentiment
 spec:
-  default:
-    predictor:
-      maxReplicas: 10
-      minReplicas: 0
-      parallelism: 1
-      custom:
-        container:
-          image: coreweave/fastai-sentiment:4
-          name: kfserving-container
-          env:
-          - name: STORAGE_URI
-            value: pvc://model-storage/sentiment
-          resources:
-            limits:
-              cpu: "3"
-              memory: 8Gi
-              nvidia.com/gpu: "1"
-            requests:
-              cpu: "1"
-              memory: 6Gi
+  predictor:
+    maxReplicas: 10
+    minReplicas: 0
+    containers:
+    - name: kfserving-container
+      image: coreweave/fastai-sentiment:4
+      env:
+      - name: STORAGE_URI
+        value: pvc://model-storage/sentiment
```

Comment:
I get this error when trying this example, node affinity seems fine however

```
$ k logs -f sentiment-predictor-default-00001-deployment-6c97454b98-9tqgc -c storage-initializer
/usr/local/lib/python3.7/site-packages/ray/autoscaler/_private/cli_logger.py:61: FutureWarning: Not all Ray CLI dependencies were found. In Ray 1.4+, the Ray CLI, autoscaler, and dashboard will only be usable via `pip install 'ray[default]'`. Please update your install command.
  "update your install command.", FutureWarning)
[I 210811 15:12:28 initializer-entrypoint:13] Initializing, args: src_uri [/mnt/pvc/sentiment/] dest_path[ [/mnt/models]
[I 210811 15:12:28 storage:50] Copying contents of /mnt/pvc/sentiment/ to local
Traceback (most recent call last):
  File "/storage-initializer/scripts/initializer-entrypoint", line 14, in <module>
    kfserving.Storage.download(src_uri, dest_path)
  File "/usr/local/lib/python3.7/site-packages/kfserving/storage.py", line 81, in download
    (_GCS_PREFIX, _S3_PREFIX, _LOCAL_PREFIX, _HTTP_PREFIX))
Exception: Cannot recognize storage type for /mnt/pvc/sentiment/
'gs://', 's3://', 'file://', and 'http(s)://' are the current available storage type.
```

Reply:
Did you download the model https://docs.coreweave.com/compass/examples/custom-sentiment#download-the-model?

Reply:
sure didn't 🤣